sdk/tests/conformance2/glsl3/tricky-loop-conditions.html

```diff
@@ -138,7 +138,7 @@
     loopCondition: 'bool((c = (a = b)[0]) + 1.0)',
     loopContents: 'b[0] += 1.0;',
     loopIterations: 3,
-    passCondition: 'c == 4.0'
+    passCondition: 'abs(c - 4.0) < 0.01'
```

Comment:
Maybe you can define a var tol = '0.01' and set it as passCondition: 'abs(c - 4.0) < ' + tol, something like that, here and below. So if we need to, we can easily adjust tolerance in the future.